BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/RabbitMQ/EventBusRabbitMQ.cs

```diff
@@ -50,6 +51,14 @@ public void Dispose()
         _subsManager.Clear();
     }
 
+    public void StartConsuming()
+    {
+        if(_consumer != null)
```

Comment:
We need to call `CreateConsumerChannel` before because its side-effects are needed to register event handlers.

Regarding that flaw, what do you think about throwing an Exception on the else?

Reply:
> Regarding that flaw, what do you think about throwing an Exception on the else?

That's fine.